queries/gdscript/highlights.scm

```diff
@@ -123,4 +155,186 @@
   "puppetsync"
 ] @keyword
 
-"return" @keyword.return
+"func" @keyword.function
+
+[
+  "await"
+  "return"
+] @keyword.return
+
+(call (identifier) @keyword.return
+      (#eq? @keyword.return "yield"))
```

Comment:
Would yield also apply to `@keyword.coroutine`?

Reply:
Yes it would apply